cmd/juju/machine/add_test.go

```diff
@@ -177,7 +177,7 @@ func (s *AddMachineSuite) TestParamsPassedOn(c *gc.C) {
 }
 
 func (s *AddMachineSuite) TestParamsPassedOnNTimes(c *gc.C) {
-	_, err := s.run(c, "-n", "3", "--constraints", "mem=8G", "--series=jammy")
+	_, err := s.run(c, "-n", "3", "--constraints", "mem=8G", "--base=ubuntu@22.04")
```

Comment:
WDYT of adding a small test like 
```go
func (s *AddMachineSuite) TestSeriesAndBaseError(c *gc.C) {
	_, err := s.run(c, "--series=jammy", "--base=ubuntu@22.04")
	c.Assert(err, gc.ErrorMatches, "--series and --base cannot be specified together")
}
```
for the newly added error when both series and base are passed as flags?